packages/core/contracts/oracle/implementation/Staker.sol

```diff
@@ -20,18 +20,17 @@ abstract contract Staker is StakerInterface, Ownable, Lockable, MultiCaller {
      ****************************************/
 
     uint256 public emissionRate;
-    uint256 public cumulativeActiveStake;
-    uint256 public cumulativePendingStake;
+    uint256 public cumulativeStake;
     uint256 public rewardPerTokenStored;
     uint64 public lastUpdateTime;
     uint64 public unstakeCoolDown;
 
     ExpandedIERC20 public votingToken;
 
     struct VoterStake {
```

Comment:
question: does this having a nested mapping have any impact on the ability to query the contract's `voterStakes` for a given address? if you queried `VotingV2.voterStake(someAddress)` it will just return the struct from below without any field for `pendingStakes` as this is a nested mapping that cant be returned from an accessor like this, right?

Reply:
Yes, I added [this function](https://github.com/UMAprotocol/protocol/blob/d820bcaf4dd5f75b7096036e79a619571c0b272e/packages/core/contracts/oracle/implementation/Staker.sol#L272) to able to query the pendingStakes. Thinking about the subgraph among other things.